app/controllers/api/v1/application_users_controller.rb

```diff
@@ -92,6 +92,33 @@ def index
   end
 
   ###############################################################
+  # username
+  ###############################################################
+
+  api :GET, '/application_users/username/:username',
+      'Gets a single User with the specified username.'
+  description <<-EOS
+      All users of OpenStax have an associated User object.  This endpoint
+      allows querying additional data on a user when all that is known is the
+      User's username.
+
+      Admins (for Accounts only) are identified by the is_administrator boolean.
```

Comment:
This paragraph doesn't seem like it should be here (sounds like information about models in general in Accounts -- think this info is not needed here?)